packages/aws-cdk/lib/commands/migrate.ts

```diff
@@ -0,0 +1,71 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { initializeProject, availableInitTemplates } from '../../lib/init';
+import { warning } from '../logging';
+import * as nocti from '../vendor/noctilucent';
+
+/** The list of languages supported by the built-in noctilucent binary. */
+export const MIGRATE_SUPPORTED_LANGUAGES: readonly string[] = nocti.supported_languages();
+
+export async function cliMigrate(
+  inputpath: string = process.cwd() + '/../temp.txt',
```

Comment:
~~What's the chance of (eventual) name collision?~~

This hardcodes a bunch of input that should be accepted from the command line. Will this actually output things in "~/my-dir" if the cli comand was `cdk migrate -l java -inputpath ~/template.json -outputpath ~/my-dir`?

Reply:
Yup it works, these are just setting the default. We could just delete the /../temp.txt default for the input path if we wanted, I just left it in there for now